src/Symfony/Component/Finder/Finder.php

```diff
@@ -433,9 +433,14 @@ public function sort(\Closure $closure): static
      *
      * @see SortableIterator
      */
-    public function sortByName(bool $useNaturalSort = false): static
+    public function sortByName(bool $useNaturalSort = false, bool $useCaseInsensitive = false): static
```

Comment:
A list of Boolean arguments is not an expressive method signature. Alternatively, I propose that we allow the sort flag to be passed as argument; but I'm now sure our backward compatibility policy allows to modify the type from `bool` to `bool|int`.

Maybe we can add the method `sortByNameCaseInsensitive(bool $useNaturalSort = false)`.

Reply:
Thank you for your comments.
I will work on a dedicated sort function and correct the changelog.
Is this method name good enough ? `sortByNameCaseInsensitive(bool $useNaturalSort = false)` It seems a bit long but it is self explanatory.
How about `sortByCaseName(bool $useNaturalSort = false)` which is closer to the php functions `strcasecmp `and `strnatcasecmp` ?

Reply:
3rd alternative, the `Finder::sort()` accepts a closure, but could also accept a value from a dedicated Enum. 

@symfony/mergers I'm not sure if we can add a 2nd allowed type to an existing argument. From `\Closure $arg` to `MyEnum|\Closure $arg`

Reply:
Meanwhile I have moved the code to `sortByCaseName(bool $useNaturalSort = false)`